web-api/terraform/api/api-public.tf

```diff
@@ -23,6 +23,8 @@ resource "aws_lambda_function" "api_public_lambda" {
 resource "aws_api_gateway_rest_api" "gateway_for_api_public" {
   name = "gateway_api_public_${var.environment}_${var.current_color}"
 
+  minimum_compression_size = "1"
+
```

Comment:
Is this part of the gzip refactoring story?

Reply:
Yes!

Reply:
cool, and it's harmless to include here? is there more work for that devex task?

Reply:
It's harmless, I'm currently debugging to find the reason why the setting hasn't impacted the browser.

Reply:
Thanks. Sorry that another PR snuck through and is now conflicting with your node modules update. 😢